src/main/java/com/driving/school/service/StudentService.java

```diff
@@ -0,0 +1,22 @@
+package com.driving.school.service;
+
+import com.driving.school.model.StudentModel;
+import com.driving.school.repository.StudentRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class StudentService {
+    private final StudentRepository repo;
+    public List<StudentModel> getStudents(){
+        return repo.findAll();
+    }
+
+    public Optional<StudentModel> getStudentById(long id){
+        return repo.findById(id);
```

Comment:
`public StudentModel getStudentById(long id){`
`        return repo.findById(id).orElseThrow(()->RuntimeException("No such student in database"));`
`}`

drobna sugestia - przy tym nie trzeba by sprawdzać czy w controllerze pobraliśmy obiekt